src/lib/containers/widgets/Checkbox.tsx

```diff
@@ -9,14 +9,18 @@ export type CheckboxProps = {
   isSelectAll?: boolean
   children?: React.ReactChild
   onClick?: (event: React.SyntheticEvent<HTMLDivElement>) => void
+  disabled?: boolean
 }
 
 export const Checkbox: React.FunctionComponent<CheckboxProps> = (
   props: CheckboxProps,
 ) => {
-  const { checked: propsChecked = false, isSelectAll = false } = props
+  const {
+    checked: propsChecked = false,
+    isSelectAll = false,
+    disabled = false,
+  } = props
   const [checked, setChecked] = useState<boolean>(propsChecked)
-  const [focused, setFocused] = useState<boolean>(false)
```

Comment:
Removed 'focus' state from both checkboxes and radios. We now just look at the `:focus` state of the input element instead of relying on a class set by React.